integration_test/util/auth.js

```diff
@@ -0,0 +1,35 @@
+const puppeteer = require('puppeteer')
+
+async function authenticateKeycloak (username, password) {
```

Comment:
can we have some timeouts in the requests here? I happened to run the integration test in a poor network and it was waiting indefinitely (you know it goes to security.feedhenry.org).